s3/src/test/scala/akka/stream/alpakka/s3/S3SettingsSpec.scala

```diff
@@ -250,4 +252,20 @@ class S3SettingsSpec extends S3WireMockBase with S3ClientIntegrationSpec with Op
     settings.forwardProxy.value.credentials.value.username shouldEqual "knock-knock"
     settings.forwardProxy.value.credentials.value.password shouldEqual "whos-there"
   }
+
+  it should "skip parsing forward-proxy when optional environment overrides exist but aren't set" in {
+    @nowarn("msg=possible missing interpolator: detected an interpolated expression")
```

Comment:
This warning occurs because `$` is in the raw string however its never used for actual interpolation (it is the valid syntax for doing an environment variable override).